cmd2.py

```diff
@@ -1107,8 +1107,11 @@ def __init__(self, completekey='tab', stdin=None, stdout=None, persistent_histor
         # will be added if there is an unmatched opening quote
         self.allow_closing_quote = True
 
-        # If the tab-completion matches should be displayed in a way that is different than the actual match values,
-        # then place those results in this list. path_complete uses this to show only the basename of completions.
+        # Use this list if you are completing strings that contain a common delimiter and you only want to
```

Comment:
Ok, so your fix for some previous issues uses ``self.display_matches`` in a few cases where it previously used ``self.completion_matches`` and improves the documentation to explain why.

This seems good to me.